templates/workspace_services/guacamole/user_resources/guacamole-azure-linuxvm/terraform/linuxvm.tf

```diff
@@ -62,9 +62,12 @@ resource "azurerm_linux_virtual_machine" "linuxvm" {
     type = "SystemAssigned"
   }
 
-  tags = {
-    parent_service_id = var.parent_service_id
-  }
+  tags = merge(
+    local.tre_user_resources_tags,
+    {
+      parent_service_id = var.parent_service_id
+    }
+  )
```

Comment:
I think we don't need the parent service id anymore as that is the workspace id which you anyway include.
So this can be just
```terraform
tags = local.tre_user_resources_tags
```